source/d_threedi_model.rst

```diff
@@ -5,158 +5,158 @@ Running a 3Di model
 
 This section helps you get started with interactive simulation with the integral 3Di modeling tools. Through the 3Di portal (version 2.0) the models of your organization can be loaded and simulations can be started. During the session others can follow the simulations 'live'. Also interventions can be done in the model during the simulation. Results are presented in real-time. All the steps from adjusting the input of water to viewing the calculation results are described in this section.
 
-*The 3Di portal supports both version 1.0 as version 2.0 models. When a functionality is specifically valid for only one version, this is indicated with* **v1** *or* **v2** *.*
-
 Start the simulation
 --------------------
 
-The simulation starts by pressing the *Play* button at the bottom right of the screen. The calculation time is displayed next to the *Play* button. 
+The simulation starts by pressing the *Play* button at the top of the screen. The calculation time is displayed next to the *Play* button. 
+
+By default, the point information tool is switched on. With it you can click anywhere on the map to visualize the time series at that location. Default waterdepth and waterlevel will be shown. If the model contains groundwater that graph is also shown. The time series can all be downloaded in CSV format. The points over time that are shown are the points calculated by the 3Di calculation core and are independent from the output time step that the modeller has set.
```

Comment:
General suggestion, a newline after a period makes it more readable. As long as the text has no empty lines in between it will be rendered as a HTML paragraph.